jrcc-access-spring-boot-autoconfigure/src/main/java/ca/bc/gov/open/jrccaccess/autoconfigure/plugins/http/DocumentController.java

```diff
@@ -64,8 +66,21 @@ public DocumentController(DocumentReadyHandler documentReadyHandler, AccessPrope
 		DocumentReceivedResponse response = new DocumentReceivedResponse();
 		response.setAcknowledge(true);
 
+		PluginConfig inputConfig = new PluginConfig();
+
+		inputConfig = this.accessProperties.getInput();
+
+		if (inputConfig == null) {
```

Comment:
Lets discuss that, if we put the app up with no config, it would not register the controller, so the http input would not work at all, in case this was working the api would always return 500. I don't have a clear solution yet, but we're getting closer, i think we should default it to unknown in case the config is not here, i would also create an inputConfig bean that you inject here, you don't need the full AccessProperties.java

Reply:
thanks I think that will help simplify things quite a bit :)